cpp/src/toppra/solver/qpOASES-wrapper.cpp

```diff
@@ -0,0 +1,127 @@
+#include <toppra/solver/qpOASES-wrapper.hpp>
+
+#include <qpOASES.hpp>
+
+namespace toppra {
+namespace solver {
+struct qpOASESWrapper::Impl {
+  qpOASES::SQProblem qp;
+
+  Impl(Eigen::Index nV, Eigen::Index nC)
+    : qp (nV, nC)
+  {
+    qpOASES::Options options;
+    options.printLevel = qpOASES::PL_NONE;
+
+    qp.setOptions( options );
+  }
+};
+
+qpOASESWrapper::qpOASESWrapper (const LinearConstraintPtrs& constraints, const GeometricPath& path,
```

Comment:
In case you miss it, there are two implementation of the qpOASESWrapper in the Python codebase. One is the version with "warmup" that is significantly faster than the vanilla version.

https://github.com/hungpham2511/toppra/blob/d73003e3fb082fbc3f37671a153f35e20f67454b/toppra/solverwrapper/hot_qpoases_solverwrapper.py#L24

Reply:
I saw it and it is mentioned in the doc you wrote. You are right, we should add it but I think it is wiser to have the whole pipeline working before going further in this direction.